indexing-service/src/main/java/org/apache/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -635,7 +646,7 @@ public Optional<ScalingStats> getScalingStats()
   @Override
   public void start()
```

Comment:
Probably needs to be annotated with `@LifecycleStart`. I wonder if just annotating the interface method `TaskRunner.start()` would work so that the implementations don't need to worry about this.